sdccc/src/main/java/com/draeger/medical/sdccc/util/MessageGeneratingUtil.java

```diff
@@ -497,12 +497,25 @@ private void sendLocalizedText(final HostedServiceProxy service, final SoapMessa
             service.sendRequestResponse(message);
         } catch (final TransportException e) {
             if (e.getCause() != null && e.getCause() instanceof HttpException) {
-                LOG.debug("TransportException with HttpException cause");
                 if (((HttpException) e.getCause()).getStatusCode() == Constants.HTTP_PAYLOAD_TOO_LARGE) {
                     LOG.debug(
                             "TransportException with HttpException cause and {} status code",
                             Constants.HTTP_PAYLOAD_TOO_LARGE);
                     return;
+                } else {
+                    LOG.debug("TransportException with HttpException cause");
+                }
+            }
+            throw e;
+        } catch (final SoapFaultException e) {
+            if (e.getCause() != null && e.getCause() instanceof HttpException) {
```

Comment:
remove null check, _instance of_ returns false, when _e.getCause()_ is null

Reply:
true. I will simplify that.